README.md

```diff
@@ -339,6 +339,7 @@ Contributions are welcome. Please take a quick look at the [contribution guideli
  * [aasm.cr](https://github.com/veelenga/aasm.cr) - Easy to use finite state machine for Crystal classes
  * [accord](https://github.com/neovintage/accord) - Sharable validations for Crystal objects
  * [any_hash.cr](https://github.com/Sija/any_hash.cr) - Recursive Hash with better JSON::Any included
+ * [burocracia.cr](https://github.com/vnbrs/burocracia.cr) - burocracia.cr the dependecyless shard to validate, generate and format Brazilian burocracias such as CPF, CNPJ and CEP
```

Comment:
I would use the term `some Brazilian official documents such as ...`, _burocracia_ is the portugese for `bureaucracy`, and this shard is specific to validate some official documents, the word `bureaucracy` has a very large `meaning.`.

OTOH `CEP` isn't a document, is just the Brazilian postal code..

Reply:
Easily:
```
 - burocracia.cr the dependecyless shard to validate, generate/fetch and format Brazilian fiscal CPF, juridical CNPJ and postal CEP codes
```